pages/blog/20190831-newfeature.md

```diff
@@ -96,15 +96,27 @@ You will now add/modify these 4 files to your branch of the cli repo. Remember t
         ![](images/20190831-cli-addition.png)
 
 
-1. **Modules**: Lastly you will add your `temperature.sh` file into the modules folder. This is the file that has the source code for your new feature.
+4. **Modules**: Now you will add your `temperature.sh` file into the modules folder. This is the file that has the source code for your new feature.
```

Comment:
the number jumps back to 1

Reply:
I think that it is a rendering issue.

Reply:
@hiroTochigi  @pattanawadee88  wondering if its an issue with raw.githack because it shows correctly on github preview

Reply:
Found the problem. Its because there are two "new lines" and it then doesn't notice it as part of the above list. Delete line 96 and 105 and it'll work again.